### PR TITLE
Update bitwarden extension

### PR DESCRIPTION
**Pull request by anirudhganwal06**
## Description
Adds a command to create a new folder in Bitwarden vault.
This will contribute towards feature request: #10013 

It also fixes a bug 🐞  where we see "Search Vault" as loading fallback of BitwardenProvider in Generate Password and other commands

https://github.com/raycast/extensions/assets/46159714/ce7c4e89-c3c6-450c-971b-e10bfe8d3274

## Screencast

https://github.com/raycast/extensions/assets/46159714/6e9a70de-77a2-4322-b9f6-80ed1dd2e04c

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @anirudhganwal06 🔥 

Do you mind checking this @jomifepe

File: extensions/bitwarden/src/create-folder.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+);
+
+interface FormData {
+  name: string;
+}
+
+function CreateFolderComponent() {
+  const bitwarden = useBitwarden();
+
+  const [nameError, setNameError] = useState<string | undefined>();
```
Comment (jomifepe) [inline extensions/bitwarden/src/create-folder.tsx:24]: Would you be able to refactor this component to use the [`useForm`](https://developers.raycast.com/utilities/react-hooks/useform) hook? Even though it's just one field, you would be able to remove the `useState`, `validateName` function, and simplify other parts of the code.

File: extensions/bitwarden/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Bitwarden Changelog
 
+## [Fix] - 2024-01-06
+
+- Fixes issue with Loading Fallback
+
+## [New Command] - 2024-01-06
+
+- Adds Create Folder command
```
Comment (jomifepe) [inline extensions/bitwarden/CHANGELOG.md:9]:
```suggestion
## [New Command and Fix] - 2024-01-06

- Adds Create Folder command
- Fixes issue with Loading Fallback
```

File: extensions/bitwarden/assets/create-folder.png
Comment (jomifepe) [inline extensions/bitwarden/assets/create-folder.png:1]: Could you replace it with this icon please?

Create folder

File: extensions/bitwarden/src/search.tsx [hunk 3]
```diff
@@ -9,10 +9,11 @@ import { FavoritesProvider, useSeparateFavoriteItems } from "~/context/favorites
 import { SessionProvider } from "~/context/session";
 import { useVaultContext, VaultProvider } from "~/context/vault";
 import { Folder, Item } from "~/types/vault";
+import { VaultLoadingFallback } from "./components/searchVault/VaultLoadingFallback";
```
Comment (jomifepe) [inline extensions/bitwarden/src/search.tsx:12]:
```suggestion
import { VaultLoadingFallback } from "~/components/searchVault/VaultLoadingFallback";
```

File: extensions/bitwarden/src/create-folder.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import { Form, Action, showHUD, ActionPanel, showToast, Toast } from "@raycast/api";
+import { BitwardenProvider, useBitwarden } from "~/context/bitwarden";
+import RootErrorBoundary from "~/components/RootErrorBoundary";
+import { useState } from "react";
+import { SessionProvider } from "./context/session";
```
Comment (jomifepe) [inline extensions/bitwarden/src/create-folder.tsx:5]:
```suggestion
import { SessionProvider } from "~/context/session";
```

**Review by jomifepe — COMMENTED** [review]
Thank you for contributing, left a couple of suggestions.

File: extensions/bitwarden/assets/create-folder.png
Comment (anirudhganwal06) [reply extensions/bitwarden/assets/create-folder.png:1]: Sure, I missed that the icons were using gradient background color.

**Comment by pernielsentikaer** [comment]
Is this ready for another review @anirudhganwal06

**Comment by anirudhganwal06** [comment]
No @pernielsentikaer. I didn't get a chance to implement the changes.

**Comment by pernielsentikaer** [comment]
All good, just wanted to make sure 🙂

**Comment by anirudhganwal06** [comment]
@jomifepe  I've made the suggested changes. Please review.

**Review by jomifepe — APPROVED** [review]
LGTM! Left a few minor suggestions, feel free to merge if you agree and apply them.

Great job and thanks for contributing 🚀

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**